.github/workflows/updatecli.yaml

```diff
@@ -0,0 +1,43 @@
+---
+name: Bump versions
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - '.github/workflows/updatecli.yaml'
+      - 'updatecli.d/**'
+    branches:
+      - 'OPSEXP-**'
```

Comment:
I'm not sure I understand the trigger logic here.
Should we not trigger this =for `paths: [chats/alfresco-common/**]` as well? I thin kthere will be cases where we do not (or forget) update the updatecli config/templates.
If in doubt maybe we should make it a workflow dispatch only with mandatory inputs `targets.*.spec.versionincrement`

Reply:
`push` is just for testing changes to updatecli configuration, see that it runs only in diff mode
 https://github.com/Alfresco/alfresco-helm-charts/pull/7/files#diff-f3c1fa6e77c6c2264c2ed7ac4c908d4baee1aca7f4d2582914ec08138be8088cR30

while `workflow_dispatch` is the way I intended to use it (run in apply mode)
https://github.com/Alfresco/alfresco-helm-charts/pull/7/files#diff-f3c1fa6e77c6c2264c2ed7ac4c908d4baee1aca7f4d2582914ec08138be8088cR38

I thought to add the input for versionincrement but for now I think is enough, given that:
> I suppose that the automation is for covering trivial updates while if we do major rewrites/feature we'll bump manually within a PR.